docker-compose.yaml

```diff
@@ -17,9 +17,7 @@ services:
       retries: 5
 
   openfga:
-    build:
-      context: ./
-      dockerfile: ./Dockerfile
+    image: openfga/openfga:latest
```

Comment:
The reason for this change is so that our docs site info works as intended.

```
curl -LO https://openfga.dev/docker-compose.yaml
docker compose up
```

With the build context it'll require the developer to clone the repository first. 

Instead of using docker for local development flows, we should encourage the developer to use the developer `make targets` instead. Docker driven development is slow because of the intermediate builds necessary. If you want to use docker to test development changes locally, then I think it's fair to ask the developer to `docker build -t openfga/openfga:latest .` first, and then this docker-compose file would work fine.

Reply:
Should be fine.